src/utils/webrtc/webrtc.js

```diff
@@ -211,6 +211,12 @@ function sendCurrentMediaState() {
 		webrtc.webrtc.emit('audioOff')
 	} else {
 		webrtc.webrtc.emit('audioOn')
+
+		if (!webrtc.webrtc.isSpeaking()) {
+			webrtc.webrtc.emit('speakingStopped')
```

Comment:
Did you mean `'stoppedSpeaking'` ?

Reply:
Nope, that was named like that before me